Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/GenericComboBoxCtrl.inl

```diff
@@ -202,6 +216,39 @@ namespace AzToolsFramework
         }
     }
 
+    template<typename T>
+    QComboBox* GenericComboBoxCtrl<T>::GetComboBox()
+    {
+        return m_pComboBox;
+    }
+
+    template<typename T>
+    QToolButton* GenericComboBoxCtrl<T>::GetEditButton()
+    {
+        return m_editButton;
+    }
+
+    template<typename T>
+    void GenericComboBoxCtrl<T>::SetEditButtonCallBack(GenericEditButtonCallback<T> function)
+    {
+        m_editButtonCallback = function;
```

Comment:
The function should be moved into the m_editButtonCallback to avoid a copy operation

`m_editButtonCallback = AZStd::move(function);`